.chloggen/remove-skiptest-hostmetricsreceiver.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receivers/hostmetricsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove Skip.Test in load_scraper_test
+
+# One or more tracking issues related to the change
+issues: [10030]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
Changes to tests do not need a changelog entry. Just add a `[chore]` prefix to the PR title to skip the CI `changelog` job

Reply:
Understand! Thanks!